usmqe_tests/web/tendrl/test_import_cluster.py

```diff
@@ -26,15 +27,22 @@ def test_initial_import_cluster(log_in, testcase_end):
 #       When finished, remove following line(s)
 #       https://github.com/Tendrl/usmqe-tests/issues/33
     import time
-    time.sleep(60)
-    log_in.driver.get(pytest.config.getini("usm_web_url"))
-    home_page = get_landing_page(log_in.driver)
+    time.sleep(180)
+    # log out and log in again
+    upper_menu = UpperMenu(valid_credentials.driver)
+    upper_menu.open_user_menu().logout()
+    valid_credentials.loginpage.login_user(
+        pytest.config.getini("usm_username"),
```

Comment:
These credentials could be taken from valid_new_user fixture instead of usm.ini config after https://github.com/Tendrl/usmqe-tests/pull/65 is merged.